src/hooks/screen/useRootChannelListHook.ts

```diff
@@ -41,6 +42,28 @@ const useRootChannelListHook = () => {
     }
   };
 
+  const updateChannelList = async (channelId: string, channelType: BetterSocialChannelType) => {
+    if (!localDb) return;
+
+    try {
+      await ChannelList.updateChannelList(localDb, channelId, channelType);
+      refresh('channelList');
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const deleteChannelList = async (channelId: string) => {
+    if (!localDb) return;
+
+    try {
+      await ChannelList.deleteChannelList(localDb, channelId);
+      refresh('channelList');
+    } catch (e) {
+      console.log(e);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `updateChannelList` and `deleteChannelList` functions could be improved. Currently, if an error occurs, it's just logged to the console. It would be better to throw the error so that it can be handled by the caller or show a user-friendly message to the user.
```diff
  const updateChannelList = async (channelId: string, channelType: BetterSocialChannelType) => {
    if (!localDb) return;

    try {
      await ChannelList.updateChannelList(localDb, channelId, channelType);
      refresh('channelList');
    } catch (e) {
      console.error(e);
+       throw e;
    }
  };

  const deleteChannelList = async (channelId: string) => {
    if (!localDb) return;

    try {
      await ChannelList.deleteChannelList(localDb, channelId);
      refresh('channelList');
    } catch (e) {
      console.error(e);
+       throw e;
    }
  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->